src/app/components/connection-window/connection-window.component.spec.ts

```diff
@@ -46,9 +46,12 @@ describe('ConnectionWindowComponent', () => {
     fixture.detectChanges();
   });
 
-  it('clicking on the connect button without altering the default values should correctly work', () => {
+  it('clicking on the connect button without altering the default values should correctly work', fakeAsync(() => {
+    component['localstorageService'].clear();
```

Comment:
why accessing it like this? just use `TestBed.get(LocalstorageService)` like a normal person :D